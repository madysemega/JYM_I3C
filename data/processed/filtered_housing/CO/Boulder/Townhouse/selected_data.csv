period_begin,median_sale_price,median_list_price,median_ppsf,median_list_ppsf,homes_sold,pending_sales,new_listings,inventory,months_of_supply,median_dom,avg_sale_to_list,sold_above_list,price_drops,off_market_in_two_weeks,remote_work_share,asset_value_momentum,State,City,Property_Type
2019-01-31,342500.0,784000.0,371.5909090909091,464.77762602944233,12.0,9.0,34.0,74.0,6.2,71.0,0.9950841211880292,0.25,0.1756756756756756,0.2222222222222222,2.97,0.0,CO,Boulder,Townhouse
2019-02-28,395000.0,774950.0,461.9883040935673,424.0501277139208,17.0,13.0,39.0,92.0,5.4,67.0,0.990383757341298,0.1176470588235294,0.0869565217391304,0.3846153846153846,5.25,0.24327127707137364,CO,Boulder,Townhouse
2019-03-31,510000.0,879000.0,425.0725955474731,433.8541666666666,34.0,27.0,36.0,90.0,2.6,36.0,0.9991934178966444,0.1470588235294117,0.2444444444444444,0.3333333333333333,4.58,-0.07990615394154565,CO,Boulder,Townhouse
2019-04-30,481355.0,748000.0,416.6666666666666,426.12942612942607,29.0,30.0,38.0,96.0,3.3,33.0,1.0027383548387172,0.3793103448275861,0.2916666666666666,0.2,3.78,-0.019775278314472566,CO,Boulder,Townhouse
2019-05-31,410000.0,825000.0,414.3646408839779,436.45251396648047,29.0,23.0,49.0,113.0,3.9,38.0,0.9909092235192706,0.2758620689655172,0.4070796460176991,0.3478260869565217,3.59,-0.0055248618784528025,CO,Boulder,Townhouse
2019-06-30,495500.0,751950.0,420.9339534911271,420.8443537414966,38.0,42.0,35.0,101.0,2.7,45.0,0.9924991465389834,0.2368421052631578,0.4356435643564356,0.1904761904761904,4.09,0.015853941091920065,CO,Boulder,Townhouse
2019-07-31,540000.0,699900.0,444.3127962085308,453.37895637296833,51.0,27.0,62.0,117.0,2.3,60.0,0.9952605723060624,0.392156862745098,0.3675213675213675,0.074074074074074,3.28,0.055540406098165906,CO,Boulder,Townhouse
2019-08-31,435850.0,739250.0,372.9912259836461,453.23419773369005,30.0,31.0,34.0,108.0,3.6,40.0,0.9909010742425564,0.1666666666666666,0.3796296296296296,0.096774193548387,3.91,-0.1605210807194739,CO,Boulder,Townhouse
2019-09-30,380000.0,750000.0,444.7107930828861,442.43869209809264,24.0,31.0,32.0,99.0,4.1,56.0,0.979326314116248,0.0416666666666666,0.4646464646464646,0.1612903225806451,4.64,0.19228218280497722,CO,Boulder,Townhouse
2019-10-31,480500.0,770500.0,450.1338688085676,433.2681499347244,38.0,23.0,28.0,95.0,2.5,47.0,0.98452293905239,0.2368421052631578,0.4105263157894736,0.0869565217391304,4.32,0.012194612341398292,CO,Boulder,Townhouse
2019-11-30,408000.0,740000.0,371.64268392136665,484.8484848484849,18.0,16.0,11.0,76.0,4.2,82.0,0.9873169336581146,0.1111111111111111,0.1578947368421052,0.0625,17.37,-0.17437298174197513,CO,Boulder,Townhouse
2019-12-31,420000.0,712500.0,435.22267206477727,444.3893038241913,21.0,10.0,8.0,58.0,2.8,144.0,0.9808177976732372,0.238095238095238,0.189655172413793,0.0,5.06,0.17107827193730807,CO,Boulder,Townhouse
2020-01-31,675000.0,804950.0,426.8585131894485,501.69738118331713,31.0,14.0,34.0,58.0,1.9,96.0,0.9844109591325924,0.1935483870967741,0.2758620689655172,0.4285714285714285,5.17,-0.01921811388098793,CO,Boulder,Townhouse
2020-02-29,572450.0,795000.0,476.79365103092266,409.2382054313287,26.0,26.0,31.0,53.0,2.0,51.0,0.9904824107631414,0.3076923076923077,0.320754716981132,0.423076923076923,5.01,0.11698287910053207,CO,Boulder,Townhouse
2020-03-31,521000.0,775000.0,380.81293706293707,422.21283783783775,30.0,21.0,22.0,44.0,1.5,34.0,0.988586926651316,0.1666666666666666,0.4545454545454545,0.1428571428571428,6.12,-0.20130451351534617,CO,Boulder,Townhouse
2020-04-30,648500.0,720000.0,444.6969696969697,443.29896907216494,9.0,7.0,19.0,54.0,6.0,30.0,0.999267276956244,0.2222222222222222,0.2777777777777777,0.2857142857142857,8.17,0.1677569914686865,CO,Boulder,Townhouse
2020-05-31,450000.0,824500.0,434.1926729986432,434.7783094883568,9.0,12.0,40.0,79.0,8.8,32.0,0.977409391527068,0.2222222222222222,0.430379746835443,0.25,9.84,-0.023621246408502494,CO,Boulder,Townhouse
2020-06-30,550000.0,750000.0,378.64577179093305,446.4285714285714,24.0,27.0,38.0,86.0,3.6,55.0,0.980490136335915,0.0833333333333333,0.4651162790697674,0.3333333333333333,8.82,-0.12793145684400742,CO,Boulder,Townhouse
2020-07-31,490000.0,745000.0,427.01141255787513,397.8899216681377,32.0,32.0,48.0,93.0,2.9,47.0,0.987009721634154,0.25,0.6129032258064516,0.28125,10.25,0.1277332123324142,CO,Boulder,Townhouse
2020-08-31,589500.0,750000.0,447.7124183006536,450.31958163858224,34.0,33.0,47.0,97.0,2.9,41.0,0.9871569287045971,0.088235294117647,0.5773195876288659,0.2121212121212121,11.65,0.04847881141812049,CO,Boulder,Townhouse
2020-09-30,532000.0,660000.0,439.19977081741786,420.8065458796026,28.0,29.0,44.0,104.0,3.7,40.0,0.987986022678504,0.1071428571428571,0.5288461538461539,0.2413793103448276,12.02,-0.019013650582993757,CO,Boulder,Townhouse
2020-10-31,486000.0,795000.0,410.0790513833992,447.6351351351352,31.0,33.0,24.0,89.0,2.9,47.0,0.9883209099703738,0.1612903225806451,0.6966292134831461,0.1818181818181818,13.68,-0.06630404059596973,CO,Boulder,Townhouse
2020-11-30,581000.0,744000.0,374.9764164634855,453.2846918459736,34.0,19.0,22.0,78.0,2.3,56.0,0.985604851987094,0.0588235294117647,0.4487179487179487,0.0,12.06,-0.0855996784071148,CO,Boulder,Townhouse
2020-12-31,445500.0,942000.0,408.3885732323232,518.1534152122388,20.0,20.0,9.0,57.0,2.9,68.0,0.9832231032722054,0.1,0.3508771929824561,0.05,14.21,0.08910468846003083,CO,Boulder,Townhouse
2021-01-31,474000.0,700000.0,408.6538461538461,429.96870109546165,13.0,11.0,28.0,65.0,5.0,84.0,0.9880283028070618,0.2307692307692307,0.2615384615384615,0.2727272727272727,15.08,0.0006495600976865745,CO,Boulder,Townhouse
2021-02-28,425000.0,892500.0,442.1241050119332,499.6160702043055,29.0,20.0,24.0,62.0,2.1,45.0,0.9990483223688396,0.3448275862068965,0.1935483870967741,0.2,16.06,0.08190369226449556,CO,Boulder,Townhouse
2021-03-31,499500.0,800000.0,472.1776675923079,493.556815213064,34.0,26.0,52.0,77.0,2.3,44.0,0.9953799264168264,0.3823529411764705,0.1038961038961039,0.1923076923076923,14.86,0.06797539930459928,CO,Boulder,Townhouse
2021-04-30,505000.0,834500.0,464.74358974358967,483.6792091021434,47.0,47.0,53.0,79.0,1.7,31.0,1.022978850858994,0.5531914893617021,0.2278481012658227,0.2978723404255319,14.05,-0.015744238575757907,CO,Boulder,Townhouse
2021-05-31,510000.0,880000.0,455.6516979304828,486.6502192982456,38.0,42.0,44.0,76.0,2.0,36.0,1.0207539812252628,0.4473684210526315,0.2236842105263157,0.1666666666666666,13.45,-0.019563243073719594,CO,Boulder,Townhouse
2021-06-30,545000.0,850000.0,480.3979674618605,487.7450980392157,48.0,44.0,47.0,73.0,1.5,35.0,1.019419078659211,0.5416666666666666,0.1232876712328767,0.2272727272727272,14.31,0.054309617727251736,CO,Boulder,Townhouse
2021-07-31,588750.0,834500.0,456.0772678899674,492.017964267084,44.0,37.0,60.0,91.0,2.1,36.0,1.0070203057310358,0.3863636363636363,0.1538461538461538,0.2162162162162162,12.66,-0.05062615002388404,CO,Boulder,Townhouse
2021-08-31,510000.0,842000.0,483.85951602863366,495.0958123767648,46.0,47.0,39.0,81.0,1.8,35.0,1.0138740631640584,0.5217391304347825,0.1728395061728395,0.1276595744680851,19.6,0.06091566077651778,CO,Boulder,Townhouse
2021-09-30,491250.0,995000.0,470.85320197044337,491.1081377380606,46.0,36.0,32.0,68.0,1.5,40.0,0.9973310658076104,0.1956521739130434,0.2794117647058823,0.1111111111111111,14.32,-0.02688035189416549,CO,Boulder,Townhouse
2021-10-31,495000.0,805000.0,480.58902275769753,533.9425587467363,23.0,25.0,44.0,82.0,3.6,43.0,0.9984888329282468,0.2608695652173913,0.1951219512195122,0.2399999999999999,15.3,0.020676976914485,CO,Boulder,Townhouse
2021-11-30,570000.0,885000.0,502.047056384743,519.0972222222222,32.0,25.0,21.0,72.0,2.3,47.0,0.9919191281368837,0.375,0.1666666666666666,0.2,17.24,0.04464944601504994,CO,Boulder,Townhouse
2021-12-31,530000.0,825000.0,451.38888888888886,488.78665899942496,29.0,17.0,9.0,62.0,2.1,42.0,1.0058603038649336,0.3103448275862069,0.032258064516129,0.0588235294117647,15.8,-0.10090322580645172,CO,Boulder,Townhouse
2022-01-31,436250.0,1110000.0,476.71205909485246,525.0009783842311,24.0,24.0,20.0,51.0,2.1,55.0,1.008411362779844,0.2916666666666666,0.0588235294117647,0.2083333333333333,15.76,0.05610056168705779,CO,Boulder,Townhouse
2022-02-28,755000.0,887000.0,602.8282288852826,557.3465592064476,34.0,20.0,31.0,50.0,1.5,50.0,1.0220340559177958,0.3529411764705882,0.02,0.1499999999999999,17.34,0.264554183986641,CO,Boulder,Townhouse
2022-03-31,600500.0,995700.0,599.6108949416342,546.1293743372216,32.0,30.0,48.0,60.0,1.9,29.0,1.0478443994217492,0.65625,0.0666666666666666,0.2999999999999999,16.39,-0.005337065833160515,CO,Boulder,Townhouse
2022-04-30,485000.0,1025000.0,533.4845333715816,547.6321082464801,32.0,32.0,48.0,74.0,2.3,29.0,1.063417100853723,0.6875,0.0945945945945946,0.25,15.65,-0.1102821215022941,CO,Boulder,Townhouse
2022-05-31,604750.0,1095000.0,490.3404263535954,550.8942371384412,34.0,38.0,52.0,85.0,2.5,25.0,1.0381803992967351,0.588235294117647,0.1529411764705882,0.3947368421052631,17.06,-0.08087227336342584,CO,Boulder,Townhouse
2022-06-30,686250.0,999950.0,561.6560390097525,526.6644145486821,50.0,42.0,55.0,95.0,1.9,36.0,1.0198721516382592,0.5,0.1789473684210526,0.0952380952380952,20.3,0.1454410218355724,CO,Boulder,Townhouse
2022-07-31,520000.0,925000.0,483.09529453597247,548.0015673981192,39.0,33.0,29.0,86.0,2.2,42.0,1.002492801532826,0.3333333333333333,0.3604651162790697,0.1212121212121212,17.46,-0.13987340830927286,CO,Boulder,Townhouse
2022-08-31,655000.0,899000.0,566.4462349244958,543.7219730941704,44.0,37.0,39.0,72.0,1.6,68.0,0.9832577891891175,0.0227272727272727,0.3888888888888889,0.1081081081081081,14.08,0.172535194052313,CO,Boulder,Townhouse
2022-09-30,613750.0,998000.0,548.2456140350878,541.2467338559164,32.0,30.0,39.0,70.0,2.2,40.0,0.9973036808424548,0.09375,0.4714285714285714,0.1333333333333333,19.15,-0.03213124170881643,CO,Boulder,Townhouse
2022-10-31,556000.0,1175000.0,483.57120358890046,565.0916131778883,26.0,20.0,14.0,55.0,2.1,53.0,0.9815193264398584,0.0,0.5454545454545454,0.1,19.1,-0.1179661246538457,CO,Boulder,Townhouse
2022-11-30,521250.0,1037500.0,540.7488486244092,578.351070914,12.0,11.0,11.0,47.0,3.9,70.0,0.9754629556971014,0.0833333333333333,0.4042553191489361,0.0909090909090909,17.29,0.11824038447938134,CO,Boulder,Townhouse
2022-12-31,617500.0,791750.0,497.6259400131962,531.8548387096774,16.0,11.0,9.0,37.0,2.3,62.0,0.9834256570989226,0.125,0.1891891891891892,0.0909090909090909,17.58,-0.07974664896820727,CO,Boulder,Townhouse
2023-01-31,418000.0,889950.0,477.7076020431124,567.9988371607608,16.0,13.0,22.0,37.0,2.3,86.0,0.9797740363930452,0.0,0.1621621621621621,0.2307692307692307,19.16,-0.040026727645177895,CO,Boulder,Townhouse
2023-02-28,640500.0,1295000.0,504.24963421402265,564.2472428827905,24.0,17.0,26.0,40.0,1.7,43.0,0.9851541183644098,0.1666666666666666,0.2,0.2352941176470588,17.24,0.05556125139602641,CO,Boulder,Townhouse
2023-03-31,644500.0,1162500.0,456.6747082423344,580.3788246721709,20.0,20.0,41.0,56.0,2.8,29.0,0.9971775415522998,0.2999999999999999,0.2142857142857142,0.1499999999999999,17.21,-0.09434796327783879,CO,Boulder,Townhouse
2023-04-30,666500.0,1150000.0,492.1540656205421,584.6325167037862,21.0,26.0,47.0,76.0,3.6,28.0,1.0079050387709412,0.4761904761904761,0.3289473684210526,0.3076923076923077,16.8,0.07769065537866515,CO,Boulder,Townhouse
2023-05-31,531855.0,1050000.0,523.1231523893058,538.9064143007361,44.0,41.0,52.0,82.0,1.9,33.0,1.0037509008123295,0.4090909090909091,0.3292682926829268,0.2195121951219512,16.07,0.0629255936953721,CO,Boulder,Townhouse
2023-06-30,551000.0,889900.0,534.0807174887892,595.1612903225806,31.0,33.0,66.0,111.0,3.6,30.0,1.0014990357802436,0.3870967741935483,0.2882882882882882,0.2424242424242424,17.79,0.02094643498273019,CO,Boulder,Townhouse
2023-07-31,515000.0,999900.0,558.2519777998502,582.7937095282147,32.0,37.0,33.0,99.0,3.1,34.0,0.98984195957614,0.21875,0.4242424242424242,0.1081081081081081,14.75,0.04525769143045011,CO,Boulder,Townhouse
2023-08-31,577500.0,865000.0,540.9169442455561,564.2954856361149,42.0,39.0,48.0,94.0,2.2,55.0,0.9964784222388514,0.238095238095238,0.2978723404255319,0.1282051282051281,15.16,-0.031052345972178985,CO,Boulder,Townhouse
2023-09-30,503225.0,1131250.0,502.8548123980424,558.2137161084529,30.0,35.0,43.0,94.0,3.1,39.0,0.980639340639112,0.1,0.3723404255319148,0.1428571428571428,14.43,-0.07036594481358116,CO,Boulder,Townhouse
