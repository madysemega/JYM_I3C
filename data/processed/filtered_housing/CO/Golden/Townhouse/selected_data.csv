period_begin,median_sale_price,median_list_price,median_ppsf,median_list_ppsf,homes_sold,pending_sales,new_listings,inventory,months_of_supply,median_dom,avg_sale_to_list,sold_above_list,price_drops,off_market_in_two_weeks,remote_work_share,asset_value_momentum,State,City,Property_Type
2019-01-31,568750.0,725000.0,286.8669063667411,226.6454352441613,6.0,1.0,1.0,1.0,0.2,56.0,0.9796631839075842,0.0,,1.0,0.47,0.0,CO,Golden,Townhouse
2019-02-28,427000.0,644900.0,226.6454352441613,231.81589114732068,1.0,,1.0,2.0,2.0,,1.0,0.0,0.5,,1.57,-0.20992826215231142,CO,Golden,Townhouse
2019-03-31,,,373.69293640561415,,,,,,,,,,,,2.52,0.6487997475132956,CO,Golden,Townhouse
2019-04-30,570000.0,602700.0,260.27397260273966,232.08973021230216,3.0,,3.0,5.0,1.7,4.0,1.0022669689291903,0.3333333333333333,0.2,,2.79,-0.3035084497282742,CO,Golden,Townhouse
2019-05-31,670000.0,622450.0,197.4653698791629,246.1952271715316,1.0,5.0,9.0,9.0,9.0,4.0,1.0,0.0,0.3333333333333333,1.0,2.61,-0.24131726309584767,CO,Golden,Townhouse
2019-06-30,529740.0,585000.0,243.35286164939163,245.606128886886,5.0,5.0,7.0,9.0,1.8,45.0,0.9776243001095564,0.0,0.2222222222222222,0.5999999999999999,2.28,0.23238247697968095,CO,Golden,Townhouse
2019-07-31,753750.0,645000.0,351.4253922344169,238.9540126239856,2.0,3.0,3.0,9.0,4.5,8.0,1.0,0.0,0.2222222222222222,0.6666666666666666,3.19,0.44409804697809463,CO,Golden,Townhouse
2019-08-31,774500.0,649950.0,364.0557474517189,235.75650558112733,4.0,4.0,10.0,13.0,3.3,3.0,0.9863111148307588,0.0,0.3076923076923077,0.5,3.69,0.03594036030520242,CO,Golden,Townhouse
2019-09-30,711000.0,650000.0,381.2332439678284,230.94819761695987,5.0,2.0,4.0,15.0,3.0,7.0,0.9891959121414388,0.2,0.2666666666666666,0.0,4.13,0.047183698201022306,CO,Golden,Townhouse
2019-10-31,690000.0,687500.0,261.12850928734474,254.4494596960588,4.0,2.0,3.0,15.0,3.8,46.0,0.9799428008255572,0.25,0.4666666666666666,0.0,3.2,-0.31504265847974966,CO,Golden,Townhouse
2019-11-30,461540.0,598900.0,230.15021459227464,238.0170715692712,3.0,3.0,2.0,12.0,4.0,50.0,0.9692311791888673,0.0,0.25,0.0,2.54,-0.11863237292478745,CO,Golden,Townhouse
2019-12-31,672500.0,642500.0,331.84641863981534,269.7416883150526,2.0,3.0,1.0,5.0,2.5,110.0,0.9864201934725102,0.0,,0.3333333333333333,2.08,0.44186882131612104,CO,Golden,Townhouse
2020-01-31,634364.5,761500.0,265.19189134808846,295.35729371361407,2.0,1.0,4.0,7.0,3.5,36.0,0.9922160701754384,0.5,0.1428571428571428,0.0,2.29,-0.20085956499073576,CO,Golden,Townhouse
2020-02-29,815000.0,610000.0,341.87302938585054,234.4485594611985,2.0,2.0,1.0,6.0,3.0,368.0,0.9912911016161992,0.0,0.1666666666666666,0.0,2.61,0.2891534037785157,CO,Golden,Townhouse
2020-03-31,745000.0,715000.0,383.13908795596,240.75500770416025,4.0,3.0,4.0,6.0,1.5,46.0,0.9885332897790308,0.0,,0.3333333333333333,2.77,0.12070580309959178,CO,Golden,Townhouse
2020-04-30,639950.0,699000.0,317.1334271258596,242.8571428571429,4.0,1.0,2.0,6.0,1.5,38.0,0.9708687771869972,0.0,0.1666666666666666,0.0,3.34,-0.1722759773278143,CO,Golden,Townhouse
2020-05-31,700000.0,785000.0,354.9695740365112,255.9808612440192,1.0,5.0,5.0,4.0,4.0,397.0,0.9615384615384616,0.0,,0.4,3.7,0.11930671343464416,CO,Golden,Townhouse
2020-06-30,668000.0,573000.0,350.3394877492844,260.95238095238096,6.0,3.0,3.0,4.0,0.6999999999999998,47.0,0.9911780674882212,0.3333333333333333,0.75,0.6666666666666666,4.82,-0.013043614512016077,CO,Golden,Townhouse
2020-07-31,692500.0,656000.0,401.9517795637199,264.161220043573,3.0,2.0,1.0,3.0,1.0,17.0,0.9873090203362384,0.0,0.6666666666666666,0.0,5.58,0.14732079488387884,CO,Golden,Townhouse
2020-08-31,484000.0,689000.0,409.4754653130288,262.7174970655292,1.0,1.0,2.0,3.0,3.0,27.0,0.9979422595557094,0.0,1.0,1.0,6.48,0.018717881427158023,CO,Golden,Townhouse
2020-09-30,815000.0,759000.0,319.4825558604469,251.41420490257696,1.0,4.0,3.0,2.0,2.0,4.0,1.0,0.0,0.5,0.5,6.92,-0.21977607225816487,CO,Golden,Townhouse
2020-10-31,785000.0,700000.0,321.7599521101467,279.8191068400226,3.0,2.0,3.0,2.0,0.6999999999999998,23.0,1.011559696342305,0.6666666666666666,0.5,1.0,7.35,0.007128389979121685,CO,Golden,Townhouse
2020-11-30,894500.0,720000.0,335.8441789791344,271.41512394101034,4.0,1.0,1.0,2.0,0.5,5.0,1.0007494707328466,0.5,1.0,1.0,7.29,0.04377246694817494,CO,Golden,Townhouse
2020-12-31,855000.0,639000.0,461.1685580590161,232.9691516709512,2.0,2.0,1.0,1.0,0.5,76.0,0.9896944574079666,0.0,,0.5,6.9,0.3731622785925015,CO,Golden,Townhouse
2021-01-31,567500.0,644950.0,339.3983782558152,292.4291938997821,2.0,1.0,2.0,2.0,1.0,25.0,0.996,0.0,0.5,1.0,6.61,-0.2640470120420002,CO,Golden,Townhouse
2021-02-28,609371.0,698700.0,192.68711928975551,280.5154431317843,3.0,3.0,3.0,1.0,0.2999999999999999,17.0,1.015791873349442,0.3333333333333333,,0.6666666666666666,7.65,-0.4322685916179564,CO,Golden,Townhouse
2021-03-31,725000.0,705000.0,358.91089108910893,303.26548535384586,1.0,2.0,2.0,1.0,1.0,194.0,1.0,0.0,1.0,1.0,8.46,0.8626615645719031,CO,Golden,Townhouse
2021-04-30,522500.0,720000.0,241.16379310344828,298.02631578947364,2.0,4.0,4.0,1.0,0.5,3.0,1.0781519079126256,1.0,,0.5,9.19,-0.32806777645659935,CO,Golden,Townhouse
2021-05-31,832500.0,770000.0,171.7948717948718,297.89184234647115,4.0,6.0,5.0,,,15.0,1.002115987460815,0.25,,1.0,9.17,-0.28764235466648336,CO,Golden,Townhouse
2021-06-30,635000.0,707500.0,396.6597077244259,295.1766309468984,7.0,4.0,8.0,4.0,0.5999999999999999,4.0,1.0486698518663575,0.7142857142857143,0.25,1.0,9.22,1.3089147166048671,CO,Golden,Townhouse
2021-07-31,668500.0,780000.0,401.40452791151074,318.05331179321485,6.0,5.0,5.0,4.0,0.6999999999999998,5.0,1.0751675090018338,0.8333333333333333,0.5,0.5999999999999999,8.52,0.011961941419019073,CO,Golden,Townhouse
2021-08-31,542500.0,699000.0,361.8997089146344,318.93520843797086,4.0,5.0,1.0,,,18.0,1.0246350735242833,0.5,,0.4,7.63,-0.09841647577424717,CO,Golden,Townhouse
2021-09-30,692500.0,825000.0,403.26458579299486,309.5684803001876,4.0,4.0,4.0,,,26.0,0.95919384057971,0.0,,1.0,7.59,0.11429928198178718,CO,Golden,Townhouse
2021-10-31,560000.0,839500.0,335.1286654697786,332.65988048819526,3.0,,1.0,1.0,0.2999999999999999,17.0,1.008487365870543,0.3333333333333333,,,7.77,-0.16896083297081788,CO,Golden,Townhouse
2021-11-30,,,373.69293640561415,,,,,,,,,,,,8.01,0.11507302988175216,CO,Golden,Townhouse
2021-12-31,,,373.69293640561415,,,,,,,,,,,,8.51,0.0,CO,Golden,Townhouse
2022-01-31,,,373.69293640561415,,,,,,,,,,,,8.49,0.0,CO,Golden,Townhouse
2022-02-28,859000.0,855000.0,452.10526315789474,340.0,3.0,3.0,3.0,,,3.0,1.0,0.0,,1.0,7.94,0.20983090423515582,CO,Golden,Townhouse
2022-03-31,859000.0,799000.0,426.6970907016543,343.75,7.0,5.0,6.0,1.0,0.1,14.0,1.0413474027584426,0.5714285714285714,1.0,1.0,6.84,-0.05619968296490896,CO,Golden,Townhouse
2022-04-30,600000.0,830000.0,383.0533781261665,392.6754634636199,8.0,5.0,6.0,2.0,0.2999999999999999,5.0,1.0508559085876814,0.625,,0.8,7.55,-0.10228265794763391,CO,Golden,Townhouse
2022-05-31,715000.0,754500.0,461.2903225806452,355.56209663352524,2.0,3.0,2.0,1.0,0.5,18.0,0.9838383838383838,0.0,1.0,0.6666666666666666,8.5,0.2042455410188544,CO,Golden,Townhouse
2022-06-30,810500.0,832000.0,304.22180556240846,337.8378378378378,2.0,4.0,5.0,2.0,1.0,9.0,0.9812137872540494,0.0,,0.75,9.68,-0.34049818374582785,CO,Golden,Townhouse
2022-07-31,545000.0,825000.0,310.6638418079096,377.7576307041402,3.0,4.0,4.0,2.0,0.6999999999999998,6.0,1.0038115649119,0.3333333333333333,1.0,0.75,9.84,0.021175458588814422,CO,Golden,Townhouse
2022-08-31,597500.0,749999.0,408.3519238650504,348.8140322902133,4.0,5.0,5.0,1.0,0.2999999999999999,6.0,0.987855155134074,0.5,,0.5999999999999999,9.42,0.3144494753191893,CO,Golden,Townhouse
2022-09-30,700000.0,847000.0,301.2279355333845,336.24982393511345,5.0,2.0,4.0,3.0,0.5999999999999999,22.0,0.9639915747044138,0.0,0.6666666666666666,1.0,9.14,-0.2623325177894046,CO,Golden,Townhouse
2022-10-31,689000.0,629000.0,493.6875,397.4293928631829,5.0,5.0,4.0,2.0,0.4,9.0,0.996857142857143,0.0,,0.4,10.4,0.638916719745223,CO,Golden,Townhouse
2022-11-30,460000.0,687500.0,804.1958041958042,362.70161290322574,3.0,2.0,3.0,3.0,1.0,17.0,1.0,0.0,,1.0,10.64,0.6289571929526354,CO,Golden,Townhouse
2022-12-31,690000.0,557500.0,375.2039151712888,384.71295783893487,1.0,2.0,2.0,3.0,3.0,9.0,0.9857142857142857,0.0,0.3333333333333333,1.0,10.78,-0.5334420880913538,CO,Golden,Townhouse
2023-01-31,539000.0,732500.0,745.5048409405255,479.70442804428046,5.0,4.0,10.0,9.0,1.8,9.0,0.9817449664429528,0.0,0.1111111111111111,1.0,9.28,0.9869324673762696,CO,Golden,Townhouse
2023-02-28,849999.5,765000.0,628.1377338913054,390.38101186758274,6.0,6.0,1.0,3.0,0.5,15.0,0.9923078145957548,0.0,0.6666666666666666,0.5,10.25,-0.157433058249696,CO,Golden,Townhouse
2023-03-31,562000.0,1000000.0,584.8965391679967,338.2318637742011,4.0,3.0,5.0,4.0,1.0,144.0,1.025187702265372,0.5,,0.6666666666666666,10.36,-0.06884030745204561,CO,Golden,Townhouse
2023-04-30,560000.0,831375.0,451.219512195122,320.38354216720904,3.0,3.0,2.0,3.0,1.0,5.0,1.0178654493448709,1.0,0.3333333333333333,0.6666666666666666,9.96,-0.2285481585564304,CO,Golden,Townhouse
2023-05-31,828475.0,997500.0,425.6159355495876,380.9693877551021,2.0,7.0,8.0,4.0,2.0,80.0,0.9842571428571428,0.0,,1.0,9.52,-0.056743061754968016,CO,Golden,Townhouse
2023-06-30,686500.0,950000.0,419.5509153318078,381.2425069246352,6.0,5.0,5.0,3.0,0.5,4.0,1.0295752254613983,0.8333333333333333,1.0,1.0,10.17,-0.014249983873249938,CO,Golden,Townhouse
2023-07-31,923500.0,849950.0,456.20991497065336,354.49604086963126,6.0,2.0,4.0,4.0,0.6999999999999998,13.0,1.0032108645234623,0.3333333333333333,1.0,0.0,10.41,0.0873767600050479,CO,Golden,Townhouse
2023-08-31,1157000.0,759800.0,291.95168579754613,389.6882494004796,2.0,4.0,3.0,3.0,1.5,40.0,0.9234471884901684,0.0,1.0,1.0,10.33,-0.36004966964313667,CO,Golden,Townhouse
2023-09-30,703250.0,769000.0,482.8544622018126,340.90909090909093,2.0,1.0,3.0,5.0,2.5,4.0,1.0022289875470427,0.5,0.2,1.0,9.27,0.6538848230410561,CO,Golden,Townhouse
